### PR TITLE
ROC-6797: Read access to support event for all

### PR DESCRIPTION
**Pull request by SanchitaMoj**
**Confirm that:**

- [ ] *VERSION* has been bumped (only for definition file changes)
- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)

### Change description ###

**Does this PR introduce a breaking change?**

```
- [ ] Yes
- [ ] No
```

### CODE REVIEW

File: definition/data/sheets/State.json [hunk 1]
```diff
@@ -13,5 +13,6 @@
   {"CaseTypeID": "MoneyClaimCase", "ID": "readyForTransfer", "Name": "Ready for transfer", "Description": "Ready for transfer", "DisplayOrder": 14, "TitleDisplay": "# ${previousServiceCaseReference} ${caseName}"},
   {"CaseTypeID": "MoneyClaimCase", "ID": "reconsiderationRequested", "Name": "Reconsideration requested", "Description": "Reconsideration requested", "DisplayOrder": 15, "TitleDisplay": "# ${previousServiceCaseReference} ${caseName}"},
   {"CaseTypeID": "MoneyClaimCase", "ID": "referredMediation", "Name": "Referred to mediation", "Description": "Mediation referred", "DisplayOrder": 16, "TitleDisplay": "# ${previousServiceCaseReference} ${caseName}"}
```
Comment (ckalista) [inline definition/data/sheets/State.json]: Missing comma at the end?

File: definition/data/sheets/AuthorisationCaseEvent.json [hunk 2]
```diff
@@ -234,6 +234,10 @@
   {"CaseTypeID": "MoneyClaimCase", "CaseEventID": "SYSTEM_MAINTENANCE", "UserRole": "caseworker-cmc-systemupdate", "CRUD": "R"},
   {"CaseTypeID": "MoneyClaimCase", "CaseEventID": "SYSTEM_MAINTENANCE", "UserRole": "caseworker-cmc-legaladvisor", "CRUD": "R"},
   {"CaseTypeID": "MoneyClaimCase", "CaseEventID": "SYSTEM_MAINTENANCE", "UserRole": "caseworker-cmc-judge", "CRUD": "R"},
+  {"CaseTypeID": "MoneyClaimCase", "CaseEventID": "SupportUpdate", "UserRole": "caseworker-cmc-legaladvisor", "CRUD": "R"},
+  {"CaseTypeID": "MoneyClaimCase", "CaseEventID": "SupportUpdate", "UserRole": "caseworker-cmc-judge", "CRUD": "R"},
+   {"CaseTypeID": "MoneyClaimCase", "CaseEventID": "SupportUpdate", "UserRole": "caseworker-cmc-solicitor", "CRUD": "R"},
```
Comment (ckalista) [inline definition/data/sheets/AuthorisationCaseEvent.json]: Please remove extra space

File: definition/data/sheets/AuthorisationCaseEvent.json [hunk 3]
```diff
@@ -234,6 +234,10 @@
...
   {"CaseTypeID": "MoneyClaimCase", "CaseEventID": "LiftStay", "UserRole": "caseworker-cmc", "CRUD": "CR"},
   {"CaseTypeID": "MoneyClaimCase", "CaseEventID": "LiftStay", "UserRole": "caseworker-cmc-anonymouscitizen", "CRUD": "C"}
+
```
Comment (ckalista) [inline definition/data/sheets/AuthorisationCaseEvent.json]: Also remove the extra line here please 👍

File: definition/data/sheets/State.json [hunk 4]
```diff
@@ -12,6 +12,7 @@
...
   {"CaseTypeID": "MoneyClaimCase", "ID": "readyForTransfer", "Name": "Ready for transfer", "Description": "Ready for transfer", "DisplayOrder": 14, "TitleDisplay": "# ${previousServiceCaseReference} ${caseName}"},
   {"CaseTypeID": "MoneyClaimCase", "ID": "reconsiderationRequested", "Name": "Reconsideration requested", "Description": "Reconsideration requested", "DisplayOrder": 15, "TitleDisplay": "# ${previousServiceCaseReference} ${caseName}"},
-  {"CaseTypeID": "MoneyClaimCase", "ID": "referredMediation", "Name": "Referred to mediation", "Description": "Mediation referred", "DisplayOrder": 16, "TitleDisplay": "# ${previousServiceCaseReference} ${caseName}"}
+  {"CaseTypeID": "MoneyClaimCase", "ID": "referredMediation", "Name": "Referred to mediation", "Description": "Mediation referred", "DisplayOrder": 16, "TitleDisplay": "# ${previousServiceCaseReference} ${caseName}"},
+  {"CaseTypeID": "MoneyClaimCase", "ID": "proceedsInCaseman", "Name": "Proceeds in Caseman", "Description": "Rest of the journey will be proceeded in caseman", "DisplayOrder": 16, "TitleDisplay": "# ${previousServiceCaseReference} ${caseName}"}
```
Comment (ckalista) [inline definition/data/sheets/State.json]: Extra line.

File: definition/data/sheets/AuthorisationCaseEvent.json (hunk 3 above)
Comment (SanchitaMoj) [reply definition/data/sheets/AuthorisationCaseEvent.json]: done

File: definition/data/sheets/State.json (hunk 4 above)
Comment (SanchitaMoj) [reply definition/data/sheets/State.json]: done

File: definition/data/sheets/State.json [hunk 5]
```diff
@@ -12,6 +12,6 @@
...
   {"CaseTypeID": "MoneyClaimCase", "ID": "readyForTransfer", "Name": "Ready for transfer", "Description": "Ready for transfer", "DisplayOrder": 14, "TitleDisplay": "# ${previousServiceCaseReference} ${caseName}"},
   {"CaseTypeID": "MoneyClaimCase", "ID": "reconsiderationRequested", "Name": "Reconsideration requested", "Description": "Reconsideration requested", "DisplayOrder": 15, "TitleDisplay": "# ${previousServiceCaseReference} ${caseName}"},
-  {"CaseTypeID": "MoneyClaimCase", "ID": "referredMediation", "Name": "Referred to mediation", "Description": "Mediation referred", "DisplayOrder": 16, "TitleDisplay": "# ${previousServiceCaseReference} ${caseName}"}
-
+  {"CaseTypeID": "MoneyClaimCase", "ID": "referredMediation", "Name": "Referred to mediation", "Description": "Mediation referred", "DisplayOrder": 16, "TitleDisplay": "# ${previousServiceCaseReference} ${caseName}"},
+  {"CaseTypeID": "MoneyClaimCase", "ID": "proceedsInCaseman", "Name": "Proceeds in Caseman", "Description": "Rest of the journey will be proceeded in caseman", "DisplayOrder": 16, "TitleDisplay": "# ${previousServiceCaseReference} ${caseName}"}
```
Comment (andrewwa-kainos) [inline definition/data/sheets/State.json]: Doesn't it need an incremented DisplayOrder?

Comment (SanchitaMoj) [reply definition/data/sheets/State.json]: In that case all the display order has to be changed as others not being considered

Comment (SanchitaMoj) [reply definition/data/sheets/State.json]: igonre the above, done now

**Pull request merged**